CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Extended `AttributeValueTranslation.name` to 250 characters - #13776 by @aniav
 - Add a new `page` field on `AssignedPageAttributeValue`. First stage (migration) of a simplification of Attribute - Page relation from #13403. by michal-macioszczyk
 - Update workflow actions with poetry dependencies - #13736 by @rafiwts
+- New environment variable `PUBLIC_URL` to define URL on which Saleor is hosted (e.g., https://api.example.com/). Takes precedence over `ENABLE_SSL` and `Shop.domain` for URL generation - #13841 by @przlada
```

Comment:
This entry needs to be moved to 3.17, scope for 3.16 is closed already.

Reply:
Done